doc/source/tracking/pypistats/get_pypi_stats.ipynb

```diff
@@ -26,59 +27,73 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 16,
+   "execution_count": 2,
    "metadata": {},
    "outputs": [],
    "source": [
-    "trackpath='doc/source/tracking/pypistats/'\n",
+    "cwd = os.getcwd()\n",
+    "\n",
+    "trackpath= cwd + '/' # '/doc/source/tracking/pypistats/'\n",
     "downloadfn = \"downloads_data.csv\"\n",
     "sysdownloadfn = \"sys_downloads_data.csv\""
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 18,
+   "execution_count": 3,
    "metadata": {},
    "outputs": [],
    "source": [
-    "downloads = pypistats.overall(\"icepyx\", total=True, format=\"pandas\")\n",
+    "downloads = pypistats.overall(\"icepyx\", total=True, format=\"pandas\").drop(columns=['percent'])\n",
+    "downloads = downloads[downloads.category != \"Total\"]\n",
     "\n",
-    "try:\n",
-    "    exist_downloads = pd.read_csv(trackpath+downloadfn)\n",
-    "    dl_data = downloads.merge(exist_downloads, how='outer', \n",
-    "                              on=['category','date','downloads'], ignore_index=True)\n",
-    "except:\n",
-    "    dl_data = downloads\n",
+    "# try:\n",
+    "exist_downloads = pd.read_csv(trackpath+downloadfn, index_col=0)#.drop(columns=['percent'])\n",
+    "# exist_downloads = exist_downloads[exist_downloads.category != \"Total\"]\n",
+    "dl_data = downloads.merge(exist_downloads, how='outer', \n",
+    "                          on=['category','date','downloads']).reindex()\n",
+    "# except:\n",
+    "#     dl_data = downloads\n",
     "    \n",
     "dl_data.to_csv(trackpath+downloadfn)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
+   "execution_count": 4,
    "metadata": {},
    "outputs": [],
    "source": [
-    "sysdownloads = pypistats.system(\"icepyx\", total=True, format=\"pandas\")\n",
-    "\n",
-    "try:\n",
-    "    exist_sysdownloads = pd.read_csv(trackpath+downloadfn)\n",
-    "    dl_data = sysdownloads.merge(exist_sysdownloads, how='outer', \n",
-    "                              on=['category','date','downloads'], ignore_index=True)\n",
-    "except:\n",
-    "    dl_data = sysdownloads\n",
+    "sysdownloads = pypistats.system(\"icepyx\", total=True, format=\"pandas\").drop(columns=['percent'])\n",
+    "sysdownloads = sysdownloads[sysdownloads.category != \"Total\"]\n",
+    "# try:\n",
+    "exist_sysdownloads = pd.read_csv(trackpath+sysdownloadfn, index_col=0)#.drop(columns=['percent'])\n",
+    "# exist_sysdownloads = exist_sysdownloads[exist_sysdownloads.category != \"Total\"]\n",
+    "# exist_sysdownloads['category'] = exist_sysdownloads['category'].fillna(\"null\")\n",
+    "sysdl_data = sysdownloads.merge(exist_sysdownloads, how='outer', \n",
+    "                              on=['category','date','downloads']).reindex()\n",
+    "# except:\n",
+    "#     dl_data = sysdownloads\n",
     "    \n",
-    "dl_data.to_csv(trackpath+sysdownloadfn)"
+    "sysdl_data.to_csv(trackpath+sysdownloadfn)"
```

Comment:
I don't see the stats from this `systdl_data` dataframe being plotted anywhere. Maybe comment this line out unless you want to plot it?

```suggestion
    "sysdl_data.to_csv(trackpath+sysdownloadfn, index=False)"
```